CODE/Sources/PennyBOT/Handlers/InteractionHandler.swift

```diff
@@ -522,7 +711,12 @@ private enum ModalID {
         case add, remove, test
```

Comment:
This `ModalID` type is also to decode/encode modal-ids (`custom_id`) easier. Makes stuff much cleaner.
The Modals penny uses are just those "Text Editor"s like in /auto-pings